### PR TITLE
update Russian transaltion

### PR DESCRIPTION
**Pull request by MayanKoyote**
minor cosmetic changes to better fit into the screen frame

### CODE REVIEW

**Comment by MayanKoyote** [comment]
Also, can you tell me why in some cases the name of an element consisting of several words can be broken into lines automatically, and in some cases it continues to be written in one line? It's some kind of length limitation that doesn't work as it should?

linebreak

**Comment by Apaczer** [comment]
TY for reporting, looks like when first (or former) string is wider than Title text rectangle area then it brake check loop for available space, as such later strings only will recive "\n" newline character if wider as former string. 

I bisected the culprit in FontHelper(), working on fix so this PR can become obsolete.

**Comment by MayanKoyote** [comment]
> I bisected the culprit in FontHelper(), working on fix so this PR can become obsolete.

in this PR, only one edit (Log Viewer=) is related to this problem, the rest are either cosmetics (Explorer=) or concerns the text moving off the right edge of the screen (too many long words) which I had not previously discovered during tests.
So I think this PR is relevant in any development of events.

File: assets/translations/Russian [hunk 1]
```diff
@@ -63,11 +63,11 @@ Delete=Удалить
 Description=Описание
 Directories=Каталоги
 Directory of the box art for the selector=Каталог с миниатюрами для показа в селекторе
-Displays last launched program's output=Отображает журнал последнего запущенного приложения
+Displays last launched program's output=Сообщения последнего запущенного ярлыка
 Edit=Изменить
 emus=эмуляторы
 Exit=Выход
-Explorer=Проводник
+Explorer=Обозреватель
```
Comment (Apaczer) [inline assets/translations/Russian:70]: "Проводник" seems more accurate here (similary to Windows Explorer), why this change?

File: assets/translations/Russian [hunk 2]
```diff
@@ -63,11 +63,11 @@ Delete=Удалить
 Description=Описание
 Directories=Каталоги
 Directory of the box art for the selector=Каталог с миниатюрами для показа в селекторе
-Displays last launched program's output=Отображает журнал последнего запущенного приложения
+Displays last launched program's output=Сообщения последнего запущенного ярлыка
```
Comment (Apaczer) [inline assets/translations/Russian:66]: "Показывает сообщения" to different it from line 104?

**Review by Apaczer — CHANGES_REQUESTED** [review]
I've updated font display - it will wrap text if it's wider than the longest string or max width of Title Box (if that's the widest value of two).

File: assets/translations/Russian (hunk 1 above)
Comment (MayanKoyote) [reply assets/translations/Russian:70]: 1. Real "Проводник" especially when mentioned/compared as/with Windows Explorer is a more powerful tool. Which (as Windows Explorer can) can at least do copy, paste, etc. things. I.e. File browser (обозреватель) + file manager.
Translation variant "Обозреватель" is something simpler, which is exactly what it is, pure file browser (which is reflected in its description - Browse files and launch apps=Просмотр файлов и запуск приложений).
Dingux Commander is more suitable for the role of full powered Explorer (but in form of 2-panel commander).
2. For example, Internet Explorer (also Explorer) is/was always translated as "обозреватель".

File: assets/translations/Russian (hunk 2 above)
Comment (MayanKoyote) [reply assets/translations/Russian:66]: It might be good, but it won't fit in the allotted space. The trimming will happen here:
Показывает сообщения последнего запущенного я|рлыка
screenshot

Unfortunately, I noticed this too late (I didn’t understand at first how to make Log Viewer entry appear in the menu).
After a long search of variants (and there were many of them), the most suitable replacements seemed to me to be those that appeared here.
The meaning (even if they are the same) is not lost and they fit into the designated places.
screenshot0

File: assets/translations/Russian (hunk 1 above)
Comment (Apaczer) [reply assets/translations/Russian:70]: Ok that's fair considering it's description, remember only that it has also other functions - delete files, install packages (for zip it's only show content), install skins from gmenu2x-skin-<name>.zip file . The functionality might expand in the future, time will show. Let's go now will your aproach :+1:
